src/styles/block/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  width: calc(#{$side} * 7 + #{$gap} * 6);
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  margin: 20px auto;
+
+  @for $month-length from 28 through 31 {
+    &--month-length_#{$month-length} :nth-child(n+#{$month-length+1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    @include square ($side);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    border: 1px solid black;
+    background-color: #eee;
+    transition-duration: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties

Reply:
transition-property: background-color , transform; 
     or   transition: background-color 0.5s, transform 0.5s;
    which is better to use?